zihuns_chattingprg/GUIclient/src/net/ReadThread.java

```diff
@@ -0,0 +1,99 @@
+package net;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.Socket;
+import java.util.Iterator;
+
+import view.ClientFrame;
+
+public class ReadThread extends Thread {
+
+	Socket socket;
+	ClientFrame cf;
+
+	public ReadThread(Socket socket, ClientFrame cf) {
+		this.socket = socket;
+		this.cf = cf;
+	}
+
+	@Override
+	public void run() {
+		super.run();
+		try {
+			while (true) {
+
+				BufferedReader br = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+				String str = br.readLine();
+
+				if (str == null) {
+					System.out.println("접속끊김");
+				}
+
+				Thread.sleep(300);
+
+				// 방번호 구분자 room--
+				if (!str.equals("room--")) {
+					String wstr = "0";
+					String[] strs = str.split("room--");
+
+					wstr = strs[0] + "";
```

Comment:
wstr 에 "0"을 넣고 왜 다시 strs[0] + ""; 을 넣나요...?
```suggestion
					String[] strs = str.split("room--");
					String wstr = strs[0] + "";
```